packages/edit-site/src/components/block-editor/index.js

```diff
@@ -87,15 +96,19 @@ export default function BlockEditor( { setIsInserterOpen } ) {
 				<BlockInspector />
 			</SidebarInspectorFill>
 			<BlockTools
-				className="edit-site-visual-editor"
+				className={ classnames( 'edit-site-visual-editor', {
+					'is-template-part': isTemplatePart,
```

Comment:
Right, so per @jameskoster's feedback (I was confused about this too) we probably should show the black border when there is a `is-isolated` class instead of just whenever a template part is being edited.